tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -89,7 +90,7 @@ def resolve_launch_arguments(args):
     # try to resolve launch file in package first
     if len(args) >= 2:
         try:
-            resolved = roslib.packages.find_resource(args[0], args[1])
+            resolved = cached.find_resource(pkg=args[0], resource_name=args[1])
```

Comment:
The change I made about the arguments are just a refactoring.
Using keyword arguments is good habit I think.